pkg/executor/redirect.go

```diff
@@ -288,14 +268,20 @@ func (executor *RedirectExecutor) ExecutorComQuery(ctx *proto.Context) (proto.Re
 				res, warn, err = rt.Execute(ctx)
 			}
 		}
-
 	}
 
 	executor.doPostFilter(ctx, res)
 
 	return res, warn, err
 }
 
+func isSelectDatabase(ctx *proto.Context, schemaless bool, rt runtime.Runtime) (proto.Result, uint16, error) {
```

Comment:
Suggest `isSelectDatabase` not a good name.  Maybe `isXXX` will return `bool` value, but this method logic is execute statement.

Suggest `executeStmt` or other proper name.